less/home-page.less

```
@@ -1,24 +1,56 @@
 .home {
 
+  .fun-bus-img {
+    @media @mobile {
+      width: 90%;
+      margin: 0 5%;
+    }
+  }
+
+  .fun-bus-section {
+    padding-top: 30px;
+    border-bottom: 2px dashed #C8C8C8;
+    @media @mobile {
+      width: 90%;
+      margin: 0 5%;
+
```

```suggestion
  .fun-bus-img {
    @media @mobile {
      width: 90%;
      margin: 0 5%;
    }
  }

  .fun-bus-section {
    padding-top: 30px;
    border-bottom: 2px dashed #C8C8C8;
    @media @mobile {
      width: 90%;
      margin: 0 5%;
```
Great job on the responsive design and using percent based here!